tracing-subscriber/src/fmt/mod.rs

```diff
@@ -57,7 +57,143 @@
 //!     .finish();
 //! ```
 //!
-//! You can find the configuration methods for [`FmtCollector`] in [`fmtBuilder`].
+//! You can find the configuration methods for [`FmtCollector`] in
+//! [`fmtBuilder`].
```

Comment:
Not introduced in this change, but I'd like to remove usage of the second-person tense in documentation. How about:
```suggestion
//! The configuration methods for [`FmtCollector`] can be found in
//! [`fmtBuilder`].
```

Reply:
+1. I always try to avoid second person language in API docs. But, I think there's some contention over this point — some people seem to find it "friendlier".